frontend/src/app/components/ExperimentRowCard.js

```diff
@@ -156,7 +155,7 @@ export default class ExperimentRowCard extends React.Component {
 
 ExperimentRowCard.propTypes = {
   experiment: React.PropTypes.object.isRequired,
-  hasAddon: React.PropTypes.bool.isRequired,
+  hasAddon: React.PropTypes.any,
```

Comment:
hasAddon is now trinary, with null indicating that we are being rendered at build time, and true and false being reserved for when we are running in the browser.